paper/paper.md

```diff
@@ -0,0 +1,150 @@
+variables, it is important to regularize. Regularization prevents
+overfitting by penalizing the negative log likelihood and can be used to
+articulate prior knowledge about the parameters in a structured form. In Pyglmnet, we offer
+users the ability to combine different types of regularization with different noise
+distributions in the GLMs.
+
+Despite the attractiveness of regularized GLMs, the available tools in
+the Python data science eco-system are highly fragmented. More
+specifically,
+
+-  [statsmodels] provides a wide range of link functions but no regularization.
+-  [scikit-learn] provides elastic net regularization but only for linear models.
+-  [lightning] provides elastic net and group lasso regularization, but only for
+   linear and logistic regression.
+
+[Pyglmnet] is a response to this fragmentation. Here are is a comparison of Pyglmnet with existing toolboxes.
+
+|                    | [pyglmnet] | [scikit-learn] | [statsmodels] |   [lightning]   |   [py-glm]    | [Matlab]|   [glmnet] in R |
+|--------------------|:----------:|:--------------:|:-------------:|:---------------:|:-------------:|:-------:|:---------------:|
+| **distributions**  |            |                |               |                 |               |         |                 |
+| gaussian           |    x       |      x         |      x        |       x         |      x        |    x    |  x              |
+| binomial           |    x       |      x         |      x        |       x         |      x        |    x    |  x              |
+| poisson            |    x       |                |      x        |                 |      x        |    x    |  x              |
+| softplus           |    x       |                |               |                 |               |         |                 |
+| probit             |    x       |                |               |                 |               |         |                 |
+| gamma              |    x       |                |               |                 |               |    x    |                 |
+| **regularization** |            |                |               |                 |               |         |                 |
+| l2                 |    x       |      x         |               |       x         |               |         |                 |
+| lasso              |    x       |      x         |               |       x         |               |         |  x              |
+| group lasso        |    x       |                |               |       x         |               |         |  x              |
+| tikhonov           |    x       |                |               |                 |               |         |                 |
+
+Pyglmnet follows the same approach as in the [Friedman, J., Hastie, T., & Tibshirani, R. (2010)](https://core.ac.uk/download/files/153/6287975.pdf>) and the accompanying widely popular R package [glmnet].
```

Comment:
"Pyglmnet follows the same approach as in" -> "Pyglmnet implements the same algorithm described in"